content/blog/2020/01/2020-01-10-fosdem-is-coming.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "FOSDEM 2020 is coming"
+tags:
+- fosdem
+- event
+- community
+- outreachy
+author: olblak
+---
+
+image::/images/post-images/fosdem[Fosdem, role=center]
+
+link:https://fosdem.org/2020/[FOSDEM] 2020 is coming and with it, a lot of great folks come in town.  
+It’s always a great moment to meet Jenkins community members, share stories and get inspired.
+I hope that this year will be as great as it always been and for that, we organize a few things
+
+== Things we'll do
+
+On the Thursday 30 of January, there will be two workshops one about link:https://www.eventbrite.com/e/jenkins-pipeline-fundamentals-training-tickets-87080214265[Jenkin Pipelines] lead by Mark Waite, and a second one about link:https://www.eventbrite.com/e/workshop-cloud-native-kubernetes-first-serverless-continuous-delivery-with-jenkins-x-kubernetes-and-tickets-87082627483[JenkinsX] by Viktor Farcic. We are asking to register in advance and ask a small fee to avoid no-show people.
+
+On the Friday 31 of January, the Jenkins project will hold a link:https://www.meetup.com/jenkinsmeetup/events/267684785/[Contributor Summit]. If you want to improve Jenkins and don’t know where to start, there will be your chance.  
```

Comment:
```suggestion
On the Friday 31 of January, the Jenkins project will hold a link:https://www.meetup.com/jenkinsmeetup/events/267684785/[Contributor Summit]. If you want to improve Jenkins and don’t know where to start, there will be your chance.  
```

> If you want to improve Jenkins and don’t know where to start, there will be your chance.  

It is not a purpose of the  summit. It actually targets experienced contributors. Please see the meetup description. Maybe the text could be reused here

Reply:
I update

Reply:
I change that section